.github/workflows/CI.yml

```diff
@@ -8,7 +8,7 @@ name: CI
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     container:
       image: ubuntu:20.04
```

Comment:
You probably need to remove these two lines, otherwise, this will still run on Ubuntu 20.04, I believe.